components/Header/index.js

```diff
@@ -11,16 +11,16 @@ const Header = ({
 }) => {
   return (
     <div
-      className={`bg-${backgroundColor}-100 md:h-1/2 max-w-md mx-auto overflow-hidden md:max-w-full`}
+      className={`bg-${backgroundColor}-100 md:h-1/2 max-w-md mb-8 mx-auto overflow-hidden md:max-w-full`}
     >
-      <div className="md:flex md:justify-around">
+      <div className="md:flex md:justify-between">
         <div className="p-8 lg:p-20 flex flex-col justify-around">
           <h2 className="font-bold tracking-wide text-3xl md:text-5xl">
             {title}
           </h2>
           <p className="mt-2 text-gray-500 text-xl md:text-2xl">{caption}</p>
           {buttonCaption && (
-            <Button caption={buttonCaption} link="/categories" />
+            <Button caption={buttonCaption} link={"#categories"} />
```

Comment:
I've put a comment about this in your other PR Negin, just make sure that whichever PR goes in first you don't overwrite your changes